ci-base/Dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM centos/devtoolset-6-toolchain-centos7
+LABEL com.vfxplatform.version="CY2019"
+
+USER root
+
+RUN yum install -y \
+    bzip2-devel \
+    file \
+    make \
+    python-devel \
```

Comment:
I'm running some CI builds against these images now. This is installing Python 2.7.5, which is outside the required 2.7.9 - 2.7.latest version range.

Reply:
Indeed! Another one to build from source...

Reply:
I'm also hitting a Python related build failure using these images for CI:
https://dev.azure.com/imageworks/OpenColorIO/_build/results?buildId=193

Reply:
I just started a new build for the ci-base with a proper python install in `/usr/local` once this build finishes https://dev.azure.com/aloysbaillet/ASWF/_build/results?buildId=137&view=logs&s=96ac2280-8cb4-5df5-99de-dd2da759617d you should have a better python (I tried it with openexr...)

Reply:
OCIO is building well now with your updates. Thanks!